po/es.po

```diff
@@ -3569,7 +3568,7 @@ msgstr "Crea un fichero nuevo"
 
 #: gnucash/gnome/gnc-plugin-basic-commands.c:116
 msgid "_Open..."
-msgstr "_Abrir…"
+msgstr "_Abrir..."
```

Comment:
Again:This is neither desired nor required. We will replace 3 dots by the more modern hellipsis in the whole program.